apps/web/lib/features/manual-time/add-manual-time-modal.tsx

```diff
@@ -47,40 +51,30 @@ export function AddManualTimeModal(props: IAddManualTimeModalProps) {
 		(e: FormEvent<HTMLFormElement>) => {
 			e.preventDefault();
 
-			const timeObject = {
-				date,
-				isBillable,
-				startTime,
-				endTime,
-				teamId,
+			const startedAt = new Date(date);
+			const stoppedAt = new Date(date);
+
+			// Set time for the started date
+			startedAt.setHours(parseInt(startTime.split(':')[0]));
+			startedAt.setMinutes(parseInt(startTime.split(':')[1]));
+
+			// Set time for the stopped date
+			stoppedAt.setHours(parseInt(endTime.split(':')[0]));
+			stoppedAt.setMinutes(parseInt(endTime.split(':')[1]));
+
+			const requestData: Omit<IAddManualTimeRequest, 'tenantId' | 'employeeId' | 'logType' | 'source'> = {
+				startedAt,
+				stoppedAt,
 				taskId,
 				description,
 				reason,
-				timeDifference
+				isBillable,
+				organizationId: team?.organizationId
 			};
 
-			if (date && startTime && endTime && teamId && taskId) {
-				setLoading(true);
-				setError('');
-				const postData = async () => {
-					try {
-						const response = await api.post('/add_time', timeObject);
-						if (response.data.message) {
-							setLoading(false);
-							closeModal();
-						}
-					} catch (err) {
-						setError('Failed to post data');
-						setLoading(false);
-					}
-				};
-
-				postData();
-			} else {
-				setError(`Please complete all required fields with a ${'*'}`);
```

Comment:
@CREDO23 it was some validation message here? How it's handled now!?

Reply:
I just added it @evereq